src/rmp/src/Restructure.cpp

```diff
@@ -44,6 +47,12 @@
 #include <fstream>
 #include <sstream>
 
+// From abc which sets its include path to abc/src
+#include <stdio.h>
+#include "misc/util/abc_global.h"
+#include "generic.h"
+#include "base/main/abcapis.h"
```

Comment:
You can also use `src/base/abc/abc.h` which has a way larger API surface instead of using the command parser.

Reply:
Yes, I want to get something basic working and then move away from the command parser for a real integration.